project/.pre-commit-config.yaml

```diff
@@ -1,47 +1,32 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+# see https://github.com/VOLTTRON/volttron-core/
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.0.1
+  rev: v4.1.0
   hooks:
-  - id: check-added-large-files
-  - id: check-toml
   - id: check-yaml
+  - id: check-json
+  - id: check-toml
+  - id: check-xml
+  - id: forbid-new-submodules
   - id: end-of-file-fixer
   - id: trailing-whitespace
-- repo: https://github.com/psf/black
```

Comment:
In lieu of the 'black' hook, we should add the 'isort' hook since we promise that code formatting tool in the [README](https://github.com/VOLTTRON/copier-poetry-volttron-agent/blob/develop/README.md)

Use the instructions in  https://pycqa.github.io/isort/docs/configuration/pre-commit.html to add the 'isort' hook.